### ISSUE TITLE
Configure thumbnail style

### ISSUE THREAD

**Issue by rossbar** [sphinx-gallery/sphinx-gallery#780]
The generated thumbnails use a default background color that very often doesn't quite match with the background color of the parent theme. The gallery in the [sphinx-gallery docs](https://sphinx-gallery.github.io/stable/auto_examples/index.html#general-examples) demonstrate the issue: the thumbnails are whiter than the off-white of the sphinx-rtd-theme page background.

It's a minor style feature, but it would be nice to be able to control/configure this. FWIW replacing the following lines with `background-color: transparent;` gives me what I was aiming for. https://github.com/sphinx-gallery/sphinx-gallery/blob/d1425b498bd34be19abd21c7b758545ad668663b/sphinx_gallery/_static/gallery.css#L7-L8

**Comment by larsoner** [comment]
You can just override this in your own CSS file, see for example

https://sphinx-gallery.github.io/stable/configuration.html#stylizing-code-links-using-css
https://github.com/mne-tools/mne-python/blob/master/doc/_static/style.css#L215-L244

**Comment by rossbar** [comment]
Great, thanks - I missed these docs because I was searching for thumbnail styling/configuration rather than link styles. Feel free to close, or I'm happy to add a brief blurb to the docs if you think it would be valuable.

**Comment by larsoner** [comment]
If you missed it in our docs they probably need improvement, so a PR to improve them would be great!

**Comment by rossbar** [comment]
Making the thumbnails fully transparent ended up being more involved than I originally thought. I ended up having to change settings in three places to get the behavior I wanted:
 1. In the CSS file to make the thumbnail container itself transparent (see my original comment)
 2. Make all of the images produced by matplotlib transparent. With a little digging, I was able to accomplish this at the configuration-level by leveraging the fact that `matplotlib_scraper` accepts `plt.savefig` kwargs (e.g. `transparent=True`). I got this idea from the [test suite](https://github.com/sphinx-gallery/sphinx-gallery/blob/d1425b498bd34be19abd21c7b758545ad668663b/sphinx_gallery/tests/test_scrapers.py#L23-L29)).
 3. Make sure that any borders that result from rescaling the thumbnails are also transparent.

Step 3 required changing sphinx-gallery source code, as the background setting from rescaling was hardcoded: https://github.com/sphinx-gallery/sphinx-gallery/blob/d1425b498bd34be19abd21c7b758545ad668663b/sphinx_gallery/utils.py#L69. It's a very simple change and I don't think it should affect current users - see #781.
